lib/commands/version_commands.sh

```diff
@@ -81,23 +81,27 @@ global_command() {
 # Output from this command must be executable shell code
 shell_command() {
   if [ "$#" -lt "2" ]; then
-    echo "Usage: asdf shell <name> <version>" >&2
+    echo "Usage: asdf shell <name> <version> | --unset" >&2
```

Comment:
I'm trying to remember what syntax is typically used for mutually exclusive parameters. I think this might be clearer: `asdf shell <name> (<version>|--unset)`.

Reply:
Just see if removing spaces is sufficient

```
asdf shell                                                                            
Usage: asdf shell <name> <version>|--unset
```

Do you think surrounding parenthesis needed?

Reply:
Well, now that I'm looking into it, it seems like something is needed, but maybe `{<version>|--unset}` is the correct syntax. Here is what I found:

IBM: `{...|...}` - https://www.ibm.com/support/knowledgecenter/en/STAV45/com.ibm.sonas.doc/sonas_manpages_syntax.html
Microsoft: `(...|...)` - https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/command-line-syntax-key
docopt: `(...|...)` - http://docopt.org/

Reply:
Thank you! I'll adopt `{...|...}`

Reply:
done in b26f566